Dockerfile

```
@@ -11,6 +11,11 @@ RUN make install EFFECTIVE_VERSION=$EFFECTIVE_VERSION
 ############# base
 FROM alpine:3.16.2 AS base
 
+############# alpine-openvpn
+FROM base AS alpine-openvpn
```

```suggestion
FROM alpine:3.16.2 AS alpine-openvpn
```

One suggestion which comes down to preference. I guess we can lose the `base` image since this is the only place where it is used and directly reference the `alpine` image

---

Done!